app/build.gradle.kts

```diff
@@ -33,6 +33,12 @@ android {
                 getDefaultProguardFile("proguard-android-optimize.txt"),
                 "proguard-rules.pro"
             )
+            buildConfigField("String", "BASE_URL", "\"https://tasky.pl-coding.com/\"")
+            buildConfigField("String", "API_KEY", "\"2362653145754dc68d3dcba130e673c3\"")
+        }
+        debug {
+            buildConfigField("String", "BASE_URL", "\"https://tasky.pl-coding.com/\"")
+            buildConfigField("String", "API_KEY", "\"2362653145754dc68d3dcba130e673c3\"")
```

Comment:
Sensitive API keys shouldn't make their way into version control (especially in public repos like here 😅). Ideally, I'd put it in `local.properties` (which is excluded from version control by default) and then read it in here in this Gradle file so you can set it as build config field.